flow-nodes/api-builder-plugin-petstore/package.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "api-builder-plugin-petstore",
+	"version": "1.0.0",
+	"description": "A flow-node to connect to the petstore API",
+	"author": "",
+	"license": "ISC",
+	"keywords": [
+		"amplify",
+		"api-builder"
+	],
+	"engines": {
+		"node": ">= 12.15"
+	},
+	"main": "index.js",
+	"files": [
+		"index.js",
+		"openapi"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/axway/api-builder-examples"
+	},
+	"dependencies": {
+		"@axway/api-builder-oas-flow-node": "^1.0.0"
+	},
+	"peerDependencies": {
+		"@axway/api-builder-runtime": "^4.5.0"
+	},
+	"devDependencies": {
+		"@axway/api-builder-runtime": "^4.5.0",
```

Comment:
Meh, probably not necessary - it's a fairly hefty dependency that only gets rid of the peer dependency warning, but technically not needed.

Reply:
it's a dev-dependency and it's required because oas-plugin has a peer dependency on runtime because it requires it explicitly for config/credential stuff. It will error if I try and run the tests without it.

Reply:
Note that this probably picked up a global instance of @axway/api-builder-runtime.